MonoGame.Framework/Utilities/FileHelpers.cs

```diff
@@ -59,5 +59,23 @@ public static string ResolveRelativePath(string filePath, string relativeFile)
             // correct platform specific separator.
             return NormalizeFilePathSeparators(localPath);
         }
+
+        /// <summary>
+        /// Returns a path relative to the base path.
+        /// </summary>
+        /// <param name="basePath">The path to make relative to.  Must end with directory seperator.</param>
+        /// <param name="path">The path to be made relative to the basePath.</param>
+        /// <returns>The relative path or the original string if it is not absolute or cannot be made relative.</returns>
+        public static string GetRelativePath(string basePath, string path)

```

Comment:
If you are going to add this you need to add a unit test for it.

https://github.com/mono/MonoGame/blob/develop/Test/Framework/UtilitiesTest.cs#L45